.travis.yml

```diff
@@ -20,11 +21,17 @@ jobs:
         - scripts/checkpatch.pl --strict --codespell -g $SHA_MAIN..$SHA_PR
     - name: "sparse check"
```

Comment:
It's "Sparse/Coccicheck checks" now.

Reply:
@plbossart 
How about make it to be a generic name?
`code checks`

Reply:
sure